components/Press/PressLinks/PressLinks.css

```diff
@@ -0,0 +1,34 @@
+.logos a {
+  color: #249cbc;
+  text-decoration: none;
+}
+
+.logos .articlesGroup > h2 {
+  text-align: center;
+}
+
+.logos .articlesGroup {
+  width: 375px;
+  padding-top: 15px;
+}
+
+@media screen and (max-width: 414px) {
```

Comment:
what if we dropped this and made the above width definition at `max-width: 375px`?

Reply:
Nice catch, i agree.